react-search-component/my-app/src/SearchableList.js

```diff
@@ -0,0 +1,42 @@
+import { useState } from 'react';
+
+export default function SearchableList({ quotes }) {
+  const [filterText, setFilterText] = useState('');
+
+  return (
+    <div className="fate">
+      <SearchBar filterText={filterText} onFilterChange={setFilterText} />
+      <QuoteList quotes={quotes} filterText={filterText} />
+    </div>
+  );
+}
+
+function SearchBar({ filterText, onFilterChange }) {
+  return (
+    <form>
+      <input
+        type="text"
+        value={filterText}
+        onChange={(e) => onFilterChange(e.target.value)}
+        placeholder="Search..."
+      />
+    </form>
+  );
+}
+
+function QuoteList({ quotes, filterText }) {
+  const filteredQuotes = quotes.filter((quote) =>
+    quote.quote.toLowerCase().includes(filterText.toLowerCase())
+  );
+
+  let listQuotes;
```

Comment:
If you used a ternary to initialize this, it could be `const` instead of `let`. But that's just a stylistic preference.